docs/guides/dkms.md

```diff
@@ -101,22 +101,27 @@ MODULES="apple-bce"
 And then run `sudo mkinitcpio -P`.
 See your distro-specific instructions for configuring `apple-bce` to added to your initramfs.
 
-# Module configuration
+# Setting up the touchbar
 
-The Touchbar module offers some modes to set. In `/etc/modprobe.d/apple-tb.conf`, set `fnmode` (`options apple-ib-tb fnmode=x`) to one of the following options:
+The touchbar can be set up by running [this script](../tools/touchbar.sh) **in Linux** using `bash /path/to/script`. Make sure your macOS is updated to the latest version before running this script.
 
-- 0: Only show F1-F12
-- 1: Show media and brightness controls, use the `fn` key to switch to F1-12
-- 2: Show F1-F12, use the `fn` key to switch to media and brightness controls
-- 3: Only show media and brightness controls
-- 4: Only show the escape key
+After running this script, restart your Mac. The touchbar should be able to change modes by pressing the fn key.
+
+If you wish to change the default mode of the touchbar, run `sudo touchbar` and choose the mode you wish.
+
+In case your touchbar is unable to change modes even after running the script and restarting, you could try the following :-
+
+1. Boot into the [macOS Recovery](https://support.apple.com/en-gb/HT201314) and then restart into Linux.
+2. Unplug all the external USB keyboards and mouse and then restart into Linux, keeping them unplugged.
+
+Still if you face an issue, mention it [here](https://github.com/t2linux/wiki/issues) or on the [discord](https://discord.com/invite/68MRhQu).
```

Comment:
We don't have links to the discord server anywhere else in the wiki (see 063fb41d0f3edd4f9afdc5020ab4f62903763262).

Also I would write `If you still face an issue, please open an issue [here](https://github.com/t2linux/wiki/issues)`

Reply:
e8e705b